e2e/docker-compose.yaml

```diff
@@ -36,10 +36,10 @@ services:
   prepare:
     image: node:18-alpine3.15
     working_dir: /e2e
-    user: "$UID"
+    user: '$UID'
     volumes:
-    - .:/e2e
-    command: "npm ci --omit=dev && npx run wait-on http://localhost:8080"
+      - .:/e2e
+    command: 'npm ci && npm run lint && npx run wait-on http://localhost:8080'
```

Comment:
I don't think linting belongs to the e2e tests. I would rather put it to the github workflow as a separate step. Then, I would omit installing dev dependencies here again.

Reply:
changed my mind, linting here is fine. readded the --omit-dev flag